src/components/input/input.stories.mdx

```diff
@@ -70,30 +88,33 @@ If the `rows` attribute is set, it will be used as the minimum height.
 <Canvas>
   <Story
     name="Elastic Textarea"
-    height="350px"
+    height="250px"
```

Comment:
Is this property necessary? (I ask because I notice it was removed from another example, probably because this component predates are inline helper that allowed us to set stories to `inline` by default.)

Reply:
Great question, @tylersticka!

My thought process was that, for the elastic textarea, it seemed more helpful to keep it so that you could more easily see the _elastic_ functionality in action. 

But, I'm open to opposing arguments. 😉